cmd/validator/validator.go

```diff
@@ -129,6 +130,7 @@ func getFlags() (validatorConfig, error) {
 		groupOutputPtr      = flag.String("groupby", "", "Group output by filetype, directory, pass-fail. Supported for Standard and JSON reports")
 		quietPtr            = flag.Bool("quiet", false, "If quiet flag is set. It doesn't print any output to stdout.")
 		globbingPrt         = flag.Bool("globbing", false, "If globbing flag is set, check for glob patterns in the arguments.")
+		formatPtr           = flag.Bool("format", false, "Format all supported file types in place. Only json is supported currently.")
```

Comment:
Thinking ahead to when multiple types are supported:

* Users need to be able to specify which config types they want to format. All can be the default but needs to be configurable
* Need to think about how we'll allow users to specify formatting settings. I know you had identified it in future work but I think it needs to be part of this PR. I don't think we'll be able to specify this on the command line, will probably have to be a config file (or multiple config files)